mobile/openapi/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 dependencies:
   http: '>=0.13.0 <0.14.0'
-  intl: '^0.18.0'
+  intl: '^0.17.0'
```

Comment:
Is this change intentional?

Reply:
It is not. It was made automatically once I ran